### PR TITLE
Fix correct previous value

### PR DESCRIPTION
**Pull request by nemisj**
### Pull Request check-list

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does your issue contain a link to existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Have you added an entry under `Future` in the changelog?

### Description of change

Previous method of the instance was working incorrectly

### CODE REVIEW

File: test/unit/instance/previous.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+/* jshint -W030 */
+var chai = require('chai')
+  , expect = chai.expect
+  , Support   = require(__dirname + '/../support')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , current   = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Instance'), function() {
+  describe('previous', function () {
+
```
Comment (sushantdhiman) [inline test/unit/instance/previous.test.js]: No extra line

File: test/unit/instance/previous.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    it('should return correct previous value', function () {
+      var Model = current.define('Model', {
+          text: {
+            type: DataTypes.STRING
+            , get: function (name) {
```
Comment (sushantdhiman) [inline test/unit/instance/previous.test.js]: no leading commas, in json object

**Comment by sushantdhiman** [comment]
@nemisj Need a PR to master before merging this

**Comment by nemisj** [comment]
@sushantdhiman what do you mean? do I have to create the same PR for master branch?

**Comment by sushantdhiman** [comment]
> what do you mean? do I have to create the same PR for master branch?

Yes, add same PR to master as well @nemisj

**Comment by nemisj** [comment]
@sushantdhiman i've added it - https://github.com/sequelize/sequelize/pull/7189

**Comment by nemisj** [comment]
@sushantdhiman how should this happen? Do I have to add it to this branch and then you will merge it?

**Comment by sushantdhiman** [comment]
Yes, you need to add changelog

**Comment by nemisj** [comment]
@sushantdhiman, added to both PRs

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# 3.30.2
```
Comment (sushantdhiman) [inline changelog.md]: Future

File: test/unit/instance/previous.test.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/* jshint -W030 */
+var chai = require('chai')
+  , expect = chai.expect
+  , Support   = require(__dirname + '/../support')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , current   = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Instance'), function() {
```
Comment (sushantdhiman) [inline test/unit/instance/previous.test.js]: Space before opening brackets `function ()`

**Comment by nemisj** [comment]
done

**Pull request merged**